docs/building_blocks_for_flexible_content.md

```diff
@@ -119,8 +119,11 @@ When added through Whitehall content can either be written in HTML or govspeak m
 
 A wrapper around the [Heading component](https://components.publishing.service.gov.uk/component-guide/heading). This block uses `content` where the component uses `text` for the title text key. This is so that headings can appear in search (only values inside a `content` key will be indexed when being published from Whitehall - see [Indexing block content in search](#indexing-block-content-in-search))
 
+If an `id` is added, the heading becomes an anchor link.
```

Comment:
The heading doesn't become an anchor link - it becomes something that can be referenced in an anchor link, so I think the wording should be changed here

Reply:
Thanks. I'll change it to:

"If an `id` is added, the heading can be referenced in an anchor link."